src/docs/product/issues/issue-details/performance-issues/slow-db-queries/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Slow DB Queries'
+sidebar_order: 30
+redirect_from:
+  - /product/issues/performance-issues/slow-db-queries/
+description: 'Learn more about Slow DB Query issues and how to diagnose and fix them.'
+---
+
+<Include name="early-adopter-note.mdx" />
+
+Slow DB Query issues are created when a particular `SELECT` SQL query in your application consistently takes longer than `1000ms` to resolve.
+
+## Detection Criteria
+
+The detector for this performance issue looks at singular database spans with the following criteria:
+
+- The span operation starts with `db` and the query is stored within the span description
+- The query is specifically a `SELECT` query
+- The span duration is at least `1000ms`
+
+Keep in mind that an issue will not be created unless the query in question is _consistently_ taking longer than `1000ms`. More specifically, the issue is created only if the same query is detected as a slow query
+in 100 events within a 24 hour period.
+
+## Span Evidence
+
+The "Span Evidence" section of a Slow DB Query issue contains the following fields:
```

Comment:
I think this might be a bit misleading to say, since the Span Evidence section does not point out the root cause of the issue, it just surfaces relevant info. The root cause of the slow query issue runs deeper than that and requires further investigation.

Reply:
Totally makes sense, but it would be good to point out the benefit of them looking at the span evidence to make this read better for the user. How about something like: You can get additional information about your Slow DB Query by looking at the following fields in the "Span Evidence" section:   ??

Reply:
That sounds good to me!